packages/adminrouter/extra/src/includes/main/common-windows.conf

```diff
@@ -0,0 +1,26 @@
+# Log info level messages and higher:
+# * state cache emits useful log messages on notice level
+# * some of the log messages that are required by test
+#   harness are not important enough to justify anything
+#   higher than info level.
+# Use a non-blocking Unix domain socket. If journald fails to read
+# the socket quickly enough, log messages may be lost, but this is
+# better than logging to a blocking connection (e.g. stderr) which
+# may cause Admin Router to hang.
+error_log C:/dcos/mesos-logs/dcos-adminrouter-error.log,facility=daemon info;
+
+# Make env vars accessible from within Lua modules.
+env SECRET_KEY_FILE_PATH;
+env AUTH_ERROR_PAGE_DIR_PATH;
+env DEFAULT_SCHEME;
+env UPSTREAM_MARATHON;
+env UPSTREAM_MESOS;
+
+env CACHE_FIRST_POLL_DELAY;
+env CACHE_POLL_PERIOD;
+env CACHE_EXPIRATION;
+env CACHE_MAX_AGE_SOFT_LIMIT;
+env CACHE_MAX_AGE_HARD_LIMIT;
+env CACHE_BACKEND_REQUEST_TIMEOUT;
+env CACHE_REFRESH_LOCK_TIMEOUT;
```

Comment:
I think we do not include the state cache at all in Agent Admin Router, as far as I remember, right? If that is the case then all these lines starting with `env CACHE_` can be removed from here.

Reply:
Let's keep this for now, as it is consistent with the Linux file.

It is true, but we should create a separate issue to move it from common -> master conf files.

Reply:
https://jira.mesosphere.com/browse/DCOS_OSS-5618